### PR TITLE
Added content for workshop section on model deployment 

### LINKED ISSUE #515: Model deployment :

### PR DESCRIPTION
**Pull request by suppathak**
Closes #515

### CODE REVIEW

File: docs/workshop/model_deployment.md [hunk 1]
```diff
@@ -1,8 +1,57 @@
...
-* Create a deployment using Seldon operator.
-* Create a route to the Seldon service.
-* Interact with an already trained model from Jupyeterlab.
-* Section where users can directly access and interact with the service using the pre-trained model and existing URL or the newly created route to the model.
+Once you complete the process of feature engineering and model training and come up with the best possible model. It is now time to deploy your model as a service. We use [Seldon](https://docs.seldon.io/projects/seldon-core/en/latest/wrappers/s2i.html) Operator on Red Hat OpenShift in order to create an endpoint which can be accessed from a terminal or jupyter notebook or can even be integrated with a bot which can directly comment on your new PullRequest.
+
+In order to deploy your model,
+
+1. Fork the repo containing the [deployment config file](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/notebooks/time-to-merge-prediction/thoth-station/thoth-ttm-deployment-config.yaml). Once you fork it, change the model path and deployment name in the config file and push it to your origin repo. You will be using this config file to create Seldon Deployment later.
```
Comment (MichaelClifford) [inline docs/workshop/model_deployment.md:7]: Do they need to fork the repo just to deploy the model? Should we assume they are already working on their own fork at this point?

Comment (oindrillac) [reply docs/workshop/model_deployment.md:7]: yeah, they should be already working off a fork at this point.

File: docs/workshop/model_deployment.md [hunk 2]
```diff
@@ -1,8 +1,57 @@
...
+
+![deployment_config](../assets/images/OpenShift_console3.png)
+
+3. Now in the Seldon Operator. Go to Seldon Deployment.
+
+![deployment_config](../assets/images/OpenShift_console4.png)
+
+Here you can see the list of different Seldon Deployments.
+
+4 In order to create a new deployment -> click on "Create SeldonDeployment". After that click on "YAML view"
```
Comment (aakankshaduggal) [inline docs/workshop/model_deployment.md:25]:
```suggestion
4. In order to create a new deployment, click on "Create SeldonDeployment". After that click on "YAML view"
```

File: docs/workshop/model_deployment.md [hunk 3]
```diff
@@ -1,8 +1,57 @@
...
+
+6. Now once you create your deployment file. Check the status of the deployment file you have created from the status bar shown in the image below.
+
+![deployment_config](../assets/images/OpenShift_console4.png)
+
+7. Once you create your deployment. Next step is to create a route to get the endpoints. In order to go to routes, click on "Networking" -> Routes.
+
+![deployment_config](../assets/images/OpenShift_console7.png)
+
+Here you can again see the list of different routes for different services. In order to create your own. Click on "Create Route".
```
Comment (aakankshaduggal) [inline docs/workshop/model_deployment.md:41]:
```suggestion
Here you can again see the list of different routes for different services. In order to create your own, click on "Create Route" and create a custom service by adding details like such:
```

File: docs/workshop/model_deployment.md (hunk 1 above)
Comment (suppathak) [reply docs/workshop/model_deployment.md]: Thanks guys. I have updated the file by removing the line from the document. Assuming that the instruction will be there is other instruction file.

**Review by aakankshaduggal — COMMENTED** [review]
/lgtm

**Pull request merged**